modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm

```diff
@@ -214,6 +214,13 @@
 	patient.client?.prefs?.safe_transfer_prefs_to_with_damage(patient)
 	patient.dna.update_dna_identity()
 	SSquirks.AssignQuirks(patient, patient.client)
+	if(patient.has_quirk(/datum/quirk/equipping/lungs/nitrogen))
+		if(!patient.get_organ_by_type(/obj/item/organ/internal/lungs/nitrogen))
+			var/lungs = patient.get_organ_slot(ORGAN_SLOT_LUNGS)
+			qdel(lungs)
+			var/obj/item/organ/target_organ = new /obj/item/organ/internal/lungs/nitrogen
+			target_organ.Insert(patient, special = TRUE)
```

Comment:
This feels very snowflakey but I think you discussed this with @Arturlang so he might be aware of a better way?

Reply:
They didnt seem to have issue with it